keyboards/ymdk/ymdk_sp64/info.json

```
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "ymdk_sp64",
+    "url": "",
+    "maintainer": "Nathan Walston",
```

This field is meant to be GitHub username(s) rather than real names.

```suggestion
    "maintainer": "walston",
```